wipac_dev_tools/prometheus.py

```diff
@@ -0,0 +1,55 @@
+from functools import partialmethod
+
+from prometheus_client import (
+    Counter,
+    Gauge,
+    Summary,
+    Histogram,
+    Info,
+    Enum,
+)
+from prometheus_client import disable_created_metrics
+
+# https://prometheus.github.io/client_python/instrumenting/#disabling-_created-metrics
+disable_created_metrics()
```

Comment:
Will this be triggered for every use of the wipac-dev-tools package, because of the `__init__.py` imports? A `try-except` around the `prometheus_client`-import that includes `disable_created_metrics()`, would fix this.

Reply:
I think it's set up so it only happens if you import this specific module.  Notice I didn't put an import in `__init__.py`.  If you're trying to use this module and don't have `prometheus_client` installed, you have problems 😄.

Reply:
That's right, looks good